supervision/utils/fps.py

```diff
@@ -0,0 +1,34 @@
+import time
+from collections import deque
+
+
+class FpsMonitor:
+    def __init__(self, sample_size: int = 30):
+        """
+        Initialize class to measure Frame per second for latency benchmark.
+        Args:
+            sample_size (int): Maximum observation to measure latency benchmark
+        Examples:
+            ```python
+            >>> import supervision as sv
+
+            >>> fps_monitor = sv.FpsMonitor()
+            >>> while True:
+            ...      (...)
+            ...      fps_monitor.tick()
+            ...      fps = fps_monitor()
+            ```
+        """
+        self.all_timestamps = deque(maxlen=sample_size)
+
+    def __call__(self, last_only=False) -> float:
```

Comment:
Hi there, Great job on the FpsMonitor class! This addition looks good especially for latency benchmarking.

I see that the `last_only` parameter in the` __call__` method is not being used. Is this a placeholder for future functionality, or can it be removed? Thanks for your work on this feature! 🚀

Reply:
Hi @artyaltanzaya, thanks for pointing it out. I removed it.